### ISSUE TITLE
Xiaomi Aqara single key wired wall switch (QBKG11LM) genOnOff generates unnecessary single event

### ISSUE THREAD

**Issue by jakliimre** [Koenkk/zigbee2mqtt#4814]
I could successfully setup the "Xiaomi Aqara single key wired wall switch (QBKG11LM)" to detached mode, but that had two annoying side effects:
1. If I generate a 'double' action, zigbee2mqtt always sends a 'single' action after the 'double' action:
[https://pastebin.com/HXXTheSX](url)
2. The other problem is that exactly in every 5 minutes a 'single' action sent out by zigbee2mqtt:
[https://pastebin.com/Kue7FpH4](url)

It seems like the processing of the 'attributeReport' in the 'genOnOff' cluster (xiaomi_on_off_action: in fromZigbee.js) causes the problem. If the payload contains a msg.data['61440'] it is ignored, but sometimes that part is missing, so two action messages were generated:
[https://pastebin.com/5qcCq6RK](url)

When the relay in "decoupled mode" the switch action reported only through the 'genMultistateInput' attribute report:
[https://pastebin.com/tFQknyin](url)

I removed the msg.data['61440'] condition in fromZigbee.js in the "xiaomi_on_off_action:" case to ignore 'genOnOff' cluster in all cases. The status change was reported reliably through the 'genMultistateInput' cluster both in "normal" and in "decoupled" mode. I tried also this version with Xiaomi QBKG12LM when only the left relay was detached. Both left and right part worked well. The double action was reported only by the decoupled switch, which is fine.

Zigbee2MQTT version: Zigbee2MQTT version 1.15.0 (commit #ed8b4e5)
Adapter hardware: CC1352P-2
Adapter firmware version: 20200925

Kind regards, Imre

**Comment by jakliimre** [comment]
UPDATE: The modification worked well with the switches having neutral line, but it blocked the operation of the Xiaomi QBKG04LM (probably also the QBKG03LM). 

So I split the processing, depending on the model of the switch.

**Comment by jakliimre** [comment]
Meanwhile updated the system, but the problem still persist with the original code:
Adapter hardware: CC1352P-2
Coordinator firmware: zStack3x0 20201026
Zigbee2MQTT version 1.16.1 (commit #cb9699e)

Problem: Double clicking on the Xiaomi QBKG11LM or QBKG12LM switches (**set to detached mode**) are sending a 'single' event after the 'double' event casusing control problems.. The double click event is used to switch on the light in the attic (it controls the physical relay in the switch) while the same switch is used to control the light on the upstairs hallway with 'single' click event. Now when I switch the attic light the upstairs hallway lights are also switched. 

As I wrote earlier I tracked down the problem: the 'fromZigbee.js' in 'zigbee-herdsman-converters' generates 'single' event when errorneously processing the 'attributeReport' message in the 'genOnOff' cluster for these two Xiaomi switch models. I modified the code and it works as it should be for a month not affecting the operation of the other QBKG03LM and QBKG04LM switches.

It is important to note that this extra message is not noticeable when the switch is not in detached mode as the physical relay controlled locally.

The modification is the following: I removed the check of the msg.data(['61440']) in case of the two problematic switch models. The other models (like the QBKG03LM and QBKG04LM I have) need that filter for msg.data in order to function properly! 

```
    xiaomi_on_off_action: {
        cluster: 'genOnOff',
        type: ['attributeReport'],
        convert: (model, msg, publish, options, meta) => {
            if (['QBKG11LM', 'QBKG12LM'].includes(model.model)) {
                return;
            }
            if (['QBKG04LM', 'QBKG21LM', 'QBKG03LM', 'QBKG22LM'].includes(model.model) && msg.data['61440']) {
                return;
            }



```
